web-console/knikubevirt/Create-vm/step-5-review/wizard-review.md

```diff
@@ -7,30 +7,10 @@ Once the user has clicked this button they will be taken to the Review screen to
 
 ![user clicks review and create button](img/review-and-create-action.png)
```

Comment:
I've noticed that on some flows we use grey "?" icons and on other flows we use black "?" icons. This is a great opportunity to standardize these.